cmd/rollkit/commands/run_node_test.go

```diff
@@ -23,7 +23,7 @@ func TestParseFlags(t *testing.T) {
 		"--p2p.unconditional_peer_ids", "4,5,6",
 		"--priv_validator_laddr", "tcp://127.0.0.1:27003",
 		"--proxy_app", "tcp://127.0.0.1:27004",
-		"--rollkit.aggregator",
+		"--rollkit.aggregator=false",
```

Comment:
For the sake of adding more test coverage, can we please add another test here covering the test case for when:
1. `--rollkit.aggregator=true` explicitly
2. `--rollkit.aggregator` is not set